Visma.net/Models/InventorySummary.cs

```diff
@@ -0,0 +1,50 @@
+﻿using ONIT.VismaNetApi.Lib;
+using ONIT.VismaNetApi.Models.CustomDto;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ONIT.VismaNetApi.Models
+{
+    public class InventorySummary : DtoProviderBase, IProvideIdentificator
```

Comment:
If it doesn't provide an identificator it shouldn't inherit IProvideIdentificator